api/machine/machine.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package machineapi;
+package machine;
```

Comment:
the name changes for the proto files are to align a bit better as well as help with identifying the socket name we need to use.